traffic_ops/traffic_ops_golang/server/servers_update_status_test.go

```diff
@@ -48,8 +47,7 @@ func TestGetServerUpdateStatus(t *testing.T) {
 	mock.ExpectQuery("SELECT").WillReturnRows(serverStatusRow)
 	mock.ExpectCommit()
 
-	dbCtx, _ := context.WithTimeout(context.TODO(), time.Duration(10)*time.Second)
-	tx, err := db.BeginTx(dbCtx, nil)
+	tx, err := db.BeginTx(context.Background(), nil)
```

Comment:
you've eliminated the timeout here, is that going to be alright?

Reply:
It's a test. `go test` has a built in timeout and the user can set it from the command line with the `-timeout` flag.

`go vet` flagged this code with the message:

>the cancel function returned by context.WithTimeout should be called, not discarded, to avoid a context leak

It seemed simpler to me to just remove the timeout since the test is not testing the timeout behavior in any way.

Reply:
As long as we know the timeout isn't being used in the tests that's fine by me